config/params.php

```diff
@@ -15,5 +16,8 @@
     'yiisoft/yii-queue' => [
         'handlers' => [],
         'channel-definitions' => [],
```

Comment:
The documentation has an example of how to use separate middleware for a specific channel?

Reply:
Yes, it is.